### ISSUE TITLE
 may need to modify the rule mark_ duplicates

### ISSUE THREAD

**Issue by user-tq** [snakemake-workflows/dna-seq-varlociraptor#215]
When trying to allow this pipeline, I encountered an error
```
 bams = snakemake.input.bams
           ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'InputFiles' object has no attribute 'bams'
```
Due to network reasons, [ localized wrapper and varlocaliraptor and forced varlocaliraptor to use version 1.23.5 wrapper](https://github.com/snakemake/snakemake-wrappers/issues/1113#issue-1614908616), which made it relatively easy for me to find the problem.（Or maybe this is a problem created by me. Sorry, I'm not an expert in this field）

I insert in the picard/markduplicates/wrapper.py
```
print(snakemake.__dict__)
print(snakemake.input.__dict__)
```
![image](https://user-images.githubusercontent.com/72878144/224491411-2f9bb7f6-3e77-4c64-aaeb-f6eb239cbecb.png)

i think is wrong to write
```
rule mark_duplicates:
    input:
        lambda wc: "results/mapped/{sample}.bam"
        if units.loc[wc.sample, "umis"].isnull().any()
        else "results/mapped/{sample}.annotated.bam",
```
maybe it should be

```
rule mark_duplicates:
    input:
        bams=lambda wc: "results/mapped/{sample}.bam"
        if units.loc[wc.sample, "umis"].isnull().any()
        else "results/mapped/{sample}.annotated.bam",
```

**Comment by dlaehnemann** [comment]
Yes, you are right that the wrapper changed and the workflow here doesn't reflect that change in the input specification. Would you feel comfortable submitting a pull request with your suggested change (which should indeed fix this)?

**Comment by dlaehnemann** [comment]
Closed by #217 .
